mesonbuild/backend/vs2010backend.py

```diff
@@ -96,6 +97,8 @@ def split_o_flags_args(args):
 def generate_guid_from_path(path, path_type):
     return str(uuid.uuid5(uuid.NAMESPACE_URL, 'meson-vs-' + path_type + ':' + str(path))).upper()
 
+def detectGDK(platform, ignore_case):
```

Comment:
detect_Microsoft_GDK would probable be a better function name and please add types to to the function